cnd/src/http_api/post.rs

```diff
@@ -0,0 +1,157 @@
+use crate::{
+    asset,
+    ethereum::ChainId,
+    halbit, hbit, herc20,
+    http_api::{DialInformation, Http},
+    identity, ledger,
+    network::{HalbitHerc20, HbitHerc20, Herc20Halbit, Herc20Hbit},
+    storage::CreatedSwap,
+    LocalSwapId, Role,
+};
+use chrono::Utc;
+
+#[derive(serde::Deserialize, Clone, Debug)]
+pub struct Body<A, B> {
+    pub alpha: A,
+    pub beta: B,
+    pub peer: DialInformation,
+    pub role: Http<Role>,
+}
+
+impl From<Body<Herc20, Halbit>> for Herc20Halbit {
```

Comment:
aye-aye

Reply:
Done, this removes the `post` module totally. Re-names `Body` to `PostBody`, this is ok for now but if we ever decide that cnd should handle another kind of POST request this will need renaming.